src/templates/template-text-classification/README.md

```diff
@@ -24,26 +24,20 @@ pip install -r requirements.txt --progress-bar off -U
 - Execute on master node
 
 ```sh
-python -m torch.distributed.launch \
+torchrun main.py \
   --nproc_per_node #:::= it.nproc_per_node :::# \
   --nnodes #:::= it.nnodes :::# \
   --node_rank 0 \
-  --master_addr #:::= it.master_addr :::# \
```

Comment:
@sayantan1410 why do you remove `--master_addr` and `--master_port` args ?

Reply:
Actually I was going through this https://pytorch.org/docs/stable/elastic/train_script.html#elastic-train-script, 
and it stated that these two args are not needed to be passes manually.

Reply:
This docs about elastic training and in any case user should provide master address/port for multi-node training. torchrun can't guess it by itself. In this elastic case, we need to provide `rdzv_endpoint`. 
For code-generator we do not need to provide elastic option, just replacing depreacted torch.distributed.launch.